.github/workflows/docs.yml

```diff
@@ -0,0 +1,33 @@
+name: Build and deploy ScrambleDB documentation
```

Comment:
Lets use something more generic like "Build & Deploy GH Pages" as this should get more over time.